CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+## Proposal for vanilla installation
+
+Most dependencies even for the extras can just be automatically installed. However, there's two exceptions: ray (as we install from URL which PyPI does not allow) and tensorflow (as we need two different versions).
+
+Installation from PyPI should work like this:
+```bash
+pip install mdp_playground
+
+# if either extra is needed:
```

Comment:
Also make this and the next comment explicitly say extras_disc is for discrete toy and extras_cont for cont. and Atari expts.